### PR TITLE
copr: add vec round_with_frac_int

### PR DESCRIPTION
**Pull request by Rustin-Liu**
### What problem does this PR solve?

Problem Summary:

Add vec round_with_frac_int.

### What is changed and how it works?

What's Changed:

Add vec round_with_frac_int.

### Related changes

None

### Check List 

Tests 

- Unit test

Side effects

None

### Release note 
- copr: add vec round_with_frac_int

### CODE REVIEW

**Comment by Rustin-Liu** [comment]
@breeswish @sticnarf  PTAL~

File: components/tidb_query_normal_expr/src/builtin_math.rs [hunk 1]
```diff
@@ -963,12 +963,6 @@ mod tests {
                 Datum::I64(-2),
                 Datum::I64(0),
             ),
-            (
```
Comment (sticnarf) [inline components/tidb_query_normal_expr/src/builtin_math.rs:966]: Why is this removed?

Comment (Rustin-Liu) [reply components/tidb_query_normal_expr/src/builtin_math.rs:966]: Already have this case.

Comment (sticnarf) [reply components/tidb_query_normal_expr/src/builtin_math.rs:966]: Oh, sorry. I didn't find it's a duplicated case.

**Comment by Rustin-Liu** [comment]
@iosmanthus  Ping~

**Comment by Rustin-Liu** [comment]
@sre-bot /run-all-tests
